src/main/java/org/syncany/gui/util/DesktopUtil.java

```diff
@@ -36,27 +36,32 @@
 import org.syncany.gui.preferences.GeneralPanel;
 import org.syncany.util.EnvironmentUtil;
 import org.syncany.util.FileUtil;
-
 import com.google.common.base.StandardSystemProperty;
+import io.undertow.util.FileUtils;
```

Comment:
Can you change that to commons.io or Guava? Other than that: Go ahead and merge. LGTM.